library/src/main/java/com/woxthebox/draglistview/ColumnProperties.java

```diff
@@ -0,0 +1,218 @@
+            Collections.addAll(this.mItemDecoration, itemDecorations);
+            return this;
+        }
+
+        /**
+         * This method is used for {@link RecyclerView#setHasFixedSize(boolean)} of items'.
+         * Set true if the items will have a fixed size and false if dynamic.
+         *
+         * @param hasFixedItemSize If the items will have a fixed or dynamic size. Default value is false.
+         *
+         * @return instance of the {@link Builder}
+         */
+        public Builder setHasFixedItemSize(boolean hasFixedItemSize) {
+            this.mHasFixedItemSize = hasFixedItemSize;
+            return this;
+        }
+
+        /**
+         * Sets background color to whole the column.
+         *
+         * @param backgroundColor Color int value. Default value is {@link Color#TRANSPARENT}.
+         *
+         * @return instance of the {@link Builder}
+         */
+        public Builder setColumnBackgroundColor(@ColorInt int backgroundColor) {
+            mColumnBackgroundColor = backgroundColor;
+            return this;
+        }
+
+        /**
+         * Sets background color to the items area only.
+         *
+         * @param backgroundColor Color int value. Default value is {@link Color#TRANSPARENT}.
+         *
+         * @return instance of the {@link Builder}
+         */
+        public Builder setItemsSectionBackgroundColor(@ColorInt int backgroundColor) {
+            mItemsSectionBackgroundColor = backgroundColor;
+            return this;
+        }
+
+        /**
+         * Sets header view that will be positioned above the column
+         *
+         * @param header View that will be positioned above the column. Default value is null.
+         *
+         * @return instance of the {@link Builder}
+         */
+        public Builder setHeader(@Nullable View header) {
+            mHeader = header;
+            return this;
+        }
+
+        /**
+         * Sets View that will act as handle to drag and drop columns. Can be null.
+         *
+         * @param columnDragView View that will act as handle to drag and drop columns. Default value is null.
+         *
+         * @return instance of the {@link Builder}
+         */
+        public Builder setColumnDrugView(@Nullable View columnDragView) {
+            mColumnDragView = columnDragView;
+            return this;
+        }
+
+        /**
+         * Builds a {@link ColumnProperties} with the settled parameters
+         *
+         * @return the {@link ColumnProperties} instance
+         */
+        public ColumnProperties build() {
+            return new ColumnProperties(mDragItemAdapter,
+                    mLayoutManager,
+                    mItemDecoration,
+                    mHasFixedItemSize,
+                    mColumnBackgroundColor,
+                    mItemsSectionBackgroundColor,
+                    mHeader,
+                    mColumnDragView);
```

Comment:
mHeader and mColumnDragView is added in the wrong order

Reply:
Changed